src/include/distributed/shard_split.h

```diff
@@ -45,4 +47,6 @@ extern void ErrorIfCannotSplitShard(SplitOperation splitOperation,
 									ShardInterval *sourceShard);
 extern void DropShardList(List *shardIntervalList);
 
+extern SplitMode LookupSplitMode(Oid shardTransferModeOid);
```

Comment:
In this file, we can now remove function 'ErrorIfCannotSplitShard' and make it static. That will also resolve the ToDo.